docs/api/authentication.md

```diff
@@ -248,7 +248,8 @@ firebase.auth().currentUser
 Updates the user's email address. See Firebase docs for more information on security & email validation. This will Promise reject is the user is anonymous.
 
 ```javascript
-firebase.auth().updateUserEmail('foo@bar.com')
+firebase.auth().currentUser
+  .updateUserEmail('foo@bar.com')
```

Comment:
This is now updateEmail as per web sdk docs, would you mind changing it? Thanks 😊